cloud_testing/kotlin_poc/src/main/kotlin/ftl/gc/GcAndroidMatrix.kt

```diff
@@ -1,17 +1,16 @@
 package ftl.gc
 
 import com.google.api.services.testing.model.AndroidMatrix
-import com.google.common.collect.Lists
+import ftl.config.Devices
 
 object GcAndroidMatrix {
 
-    fun build(
-            modelIds: String, versionIds: String, locales: String, orientations: String): AndroidMatrix {
+    fun build(deviceList: List<Devices>): AndroidMatrix {
```

Comment:
I don't think `AndroidMatrix` is what we want here. That will give us all permutations of the listed configs. 
I think want a `com.google.api.services.testing.model.AndroidDeviceList` instead